client/src/apps/Users/Users.module.css

```diff
@@ -0,0 +1,43 @@
+.container {
+  padding: 64px 30px;
+}
+
+.error {
+  display: block;
+  padding: 4px 0;
+  height: 20px;
+  color: red;
```

Comment:
Takie kolory jeśli widzisz opcję reużywalności to można dodać jako zmienną CSS w pliku `src/styles.css`